lib/rucio/daemons/auditor/__init__.py

```diff
@@ -187,10 +188,15 @@ def process_output(output, sanity_check=True, compress=True):
 
     # While converting LOST replicas to PFNs, entries that do not
     # correspond to a replica registered in Rucio are silently dropped.
+    lost_pfns = []
+    for chunk in chunks(lost_replicas, 1000):
+        lost_pfns.extend([r['rses'][rse_id][0] for r in list_replicas(chunk) if rse_id in r['rses']])
```

Comment:
I believe that this can be combined into a single statement, but I don’t have a strong preference.

Reply:
would the following do?
`lost_pfns=[r for chunk in chunks([1,2,3,4,5,6,7,8,9],2) for replicas in list_replicas(chunk) for r in replicas]`